src/HFO.cpp

```diff
@@ -328,14 +332,19 @@ status_t HFOEnvironment::step() {
   say_msg.clear();
 
   // Get the game status
-  if (recv(sockfd, &game_status, sizeof(status_t), 0) < 0) {
-    perror("[Agent Client] ERROR recieving from socket");
+  std::vector<int> full_status(3,-1);
```

Comment:
Minor comment, but you could consider using a static array rather than a re-sizable vector, since I don't think we will ever want to resize this container.
int full_status[3];